.github/workflows/release.yml

```diff
@@ -0,0 +1,47 @@
+name: Release
+
+# We want this workflow to run on master, but only after we've finished the
+# main CI workflow. This ensures that, for instance, if integration tests
+# fail on master, we don't accidentally publish broken versions of pacakges.
+#
+# See the `if` condition on the `release` job for the second half of this logic.
+on:
+  workflow_run:
+    workflows:
+      - CI
+    branches:
+      - master
+    types:
+      - completed
```

Comment:
FYI, I changed this. I think this new version is what we want!

Reply:
Makes sense to me!